src/js/tabs/setup.js

```diff
@@ -224,32 +227,19 @@ setup.initialize = function (callback) {
                 'CALIBRATING',
                 'CLI',
                 'CMS_MENU',
-                'OSD_MENU',
                 'BST',
                 'MSP',
+                'PARALYZE',
+                'GPS',
+                'RESC',
+                'RPMFILTER',
+                'REBOOT_REQUIRED',
+                'DSHOT_BITBANG',
+                'ACC_CALIBRATION',
+                'MOTOR_PROTOCOL',
+                'ARM_SWITCH',           // Needs to be the last element, since it's always activated if one of the others is active when arming
             ];
 
-            disarmFlagElements.splice(disarmFlagElements.indexOf('THROTTLE'), 0, 'RUNAWAY_TAKEOFF');
-
-            disarmFlagElements = disarmFlagElements.concat(['PARALYZE', 'GPS']);
-
-            disarmFlagElements.splice(disarmFlagElements.indexOf('OSD_MENU'), 1);
-            disarmFlagElements = disarmFlagElements.concat(['RESC']);
-            disarmFlagElements = disarmFlagElements.concat(['RPMFILTER']);
-
-            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_42)) {
```

Comment:
We currently support firmware 4.0 and up. Are we removing backwards compatibility?

Reply:
Sorry, I'm confused about backwards compatibility, misunderstand that. 
Well, I change to support from 4.0, ie. API 1.42

Reply:
Removed previous firmware support as yes backwards compatibility is often overlooked.
Firmware 4.0 is API 1.41.